src/api/search/test.js

```diff
@@ -0,0 +1,118 @@
+const request = require('supertest');
+process.env.ELASTICSEARCH_HOSTS = 'http://elasticsearch:9200';
```

Comment:
Your tests can read in `config/env.development` (see how auth's jest tests do it).